### ISSUE TITLE
Multiple powerups have a timer. How should we display them?

### ISSUE THREAD

**Issue by LuizGyro** [MarvellousSoft/hooksharks#217]
Since multiple powers have a timer to represent their duration, we need to think of a clean way of relaying this information to the player.
Maybe showing them at the same height, but horizontal to each other, and limit to a maximum of 3 simultaneous timers (considering their sizes).

**Comment by AndreMoukarzel** [comment]
Timers could be represented by concentric radial progression bars centered around the Player they belong to.

**Comment by LuizGyro** [comment]
@Detril good idea. I was thinking they could be differentiated by color, each one associated exclusively to a single powerup. The only thing that concerns me is that it may make diving confusing. Maybe limit the powerup progression bars to the lower portion of the player?
